css/styles.css

```diff
@@ -0,0 +1,73 @@
+.body {
+  background-color: #CBFDCB;
+
+}
+.header {
+  background-color: #228B22;
+  padding: 1em;
+  text-align: center;
+}
+.header__nav {
+  font-weight: bold;
+}
+.nav__links {
+  margin: 0 .5em;
+  color: #CBFDCB;
+  font-family: 'Signika', sans-serif;
+}
+.nav__links__current {
+  margin: 0 .5em;
+  color: #FFFFFF;
+  font-family: 'Signika', sans-serif;
+  text-decoration: none;
+}
+.nav__links:hover {
+  margin: 0 .5em;
+  color: #d3d3d3;
+  font-family: 'Signika', sans-serif;
+}
+.main {
+  margin: 0 5%;
+  width: auto;
+}
+.main__h1 {
+  text-align: center;
+  font-weight: bold;
+  font-family: 'Bowlby One SC', sans-serif;
+}
+.main__h2 {
+  font-family: 'Bowlby One SC', sans-serif;
+  border-left: medium solid #228B22;
+  padding: 0 0 0 .5em;
+}
+.main__copy {
+  border-left: medium solid #228B22;
+  padding: 0 0 0 1em;
+  font-family: 'Signika', sans-serif;
+}
+.main__unordered {
+  background-color: #000000;
+  color: #228B22;
+  padding: 1em 2em;
+  line-height: 1.5;
+}
+.main__listitem {
+  font-weight: bold;
+  font-family: 'Signika', sans-serif;
+}
+.footer {
+  background-color: #228B22;
+  padding: .5em .75em;
+}
+.footer__copy {
+  color: #CBFDCB;
+  font-size: .7em;
+  font-family: 'Signika', sans-serif;
+}
+.full-width {
```

Comment:
Nice use of a class as a general modifier.  👍 This will come in handy when you make your own grid layouts.